lib/process/import.go

```diff
@@ -250,3 +273,9 @@ func matchTeleportConfigPackage(teleportVersion semver.Version) pack.MatchFunc {
 		return verBase.Compare(teleportVersion) == 0
 	}
 }
+
+func matchLegacyTeleportConfigPackage() pack.MatchFunc {
+	return func(env pack.PackageEnvelope) bool {
+		return env.HasLabel(pack.PurposeLabel, pack.PurposeTeleportMasterConfig)
```

Comment:
For the legacy packages - do we need to compare them somehow, to make sure an appropriate one is selected rather than just the first one with the label? Or were they "uncomparable" before? I don't remember honestly.

Reply:
This is just a matcher implementation that selects the package which will then be evaluted in the caller. The caller searches the package with the maximum available version among the selected ones - so yes, it selects the latest which is how it should have worked before.